tests/test_config.py

```diff
@@ -310,10 +311,26 @@ def test_ssl_config(
         ssl_keyfile=tls_ca_certificate_private_key_path,
     )
     config.load()
-
     assert config.is_ssl is True
 
 
+def test_ssl_config_with_options() -> None:
+    list_options = [ssl.OP_NO_RENEGOTIATION, ssl.OP_NO_TLSv1_2]
+    config = Config(app=asgi_app, ssl_options=list_options)
+    config.load()
+    if config.ssl_options is not None:
+        assert ssl.OP_NO_RENEGOTIATION in config.ssl_options
+        assert ssl.OP_NO_TLSv1_2 in config.ssl_options
+
+
+def test_ssl_config_with_empty_options() -> None:
+    config = Config(
+        app=asgi_app,
+    )
+    config.load()
+    assert not config.ssl_options
```

Comment:
Shouldn't we check the `config.ssl` created instead?

```suggestion
    config = Config(app=asgi_app)
    config.load()
    assert not config.ssl_options
```

Reply:
agreed, removing this test as checking for config.ssl is True assertion test is already there- coverage is good.